src/megaclient.cpp

```diff
@@ -13451,6 +13482,12 @@ void MegaClient::delsync(Sync* sync, bool deletecache)
         sync->statecachetable->remove();
         delete sync->statecachetable;
         sync->statecachetable = NULL;
+
+        if (!sync->getConfig().isUpSync() && sync->localroot->node)
```

Comment:
Is it enough here to do this for down syncs only?  I think a node in a two way sync would never be !syncable?  thanks